cs/quickstart/creating-posts.texy

```diff
@@ -1,7 +1,7 @@
 Vytváření a editování příspěvků
 *******************************
 
-To je paráda! Máme super cool nový blog, lidé urputně diskutují v komentářích a my máme konečně trochu času na další programování. Přestože je Adminer skvělý nástroj, není úplně ideální pro psaní nových příspěvků na blog. Zřejmě je ten správný čas pro vytvoření jednoduchého formuláře pro přidání nových příspěvků přímo z aplikace. Pojďme na to.
+To je paráda! Máme nový blog, lidé diskutují v komentářích a my máme konečně trochu času na další programování. Přestože je Adminer skvělý nástroj, není úplně ideální pro psaní nových příspěvků na blog. Zřejmě je ten správný čas pro vytvoření jednoduchého formuláře pro přidání nových příspěvků přímo z aplikace. Pojďme na to.
```

Comment:
Nechápu, co je špatně na urputně?

Reply:
viz popis u PR -  do dokumentace nepatří žádná citově zabarvená slova a sousloví.

Reply:
Na to máme holt jiný názor.

Reply:
jak myslíš, vrátím to pane básníku

Reply:
i když s tín nesouhlasím